ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -227,14 +244,103 @@ tizen_sensor_get_type (void)
   return etype;
 }
 
+static GHashTable *tizensensors = NULL;
```

Comment:
how about adding hash-table in GstTensorSrcTIZENSENSOR struct

Reply:
If there are multiple instances of TIZENSENSORs in an application, we don't need multiple copies of hash tables. They can share a single instance of the hashtable. In fact, this hash table can be shared with ALL applications in a Tizen device.